### ISSUE TITLE
 Invalid connection, please update your connection to load complete details

### ISSUE THREAD

**Issue by jmancevska** [Azure/LogicAppsUX#5380]
### Describe the Bug with repro steps 

When using the new Logic App Generally Available Designer, I get the following error message on the custom ApiConnection:

 "Invalid connection, please update your connection to load complete details"

 The Change Connection link is available, but when I click on it, an empty pop-up appears, even though there are other connections to select from.

The Logic App works OK if I trigger it as is. However, I cannot make any changes and save them, because of the connection error message.

The error message is not there if I switch to the Legacy Designer. That is a very temporary workaround though, because the Legacy Designer won't be available from September 2024 onwards.

### What type of Logic App Is this happening in? 

Consumption (Portal)

### Are you using new designer or old designer 

New Designer

### Did you refer to the TSG before filing this issue? [https://aka.ms/lauxtsg](<https://aka.ms/lauxtsg>) 

Yes

### Workflow JSON 

*No response*

### Screenshots or Videos 

*No response*

### Browser 

Chrome, Edge

### Additional context 

*No response*

**Comment by hartra344** [comment]
Can you give us anymore information about your workflow? What actions this happening on? Which connections? Do you have RBAC access to everything?

We're going to need more information to investigate what's going on here.

**Comment by jmancevska** [comment]
The Logic App is very simple with only a very few steps.  One of the action items is to send an HTTP request to an on-prem web API endpoint.  For that step we use a Logic App Custom Connector that we have created which sends the request via on-premises data gateway.
The connection error is evident on that particular action.
I have contributor access to the logic app, the connection and to the data gateway, but I am not an admin.
This error is present even when the azure admins go to the new Logic App designer for this app.

**Comment by jmancevska** [comment]
Also, I wanted to mention that if I make a change and try to save it, I am getting this error:
Workflow Serialization Error

**Comment by pgferrero** [comment]
Getting the same issue - Invalid connection, please update your connection to load complete details, then error is Workflow Serialization Error.  This is real, folks.  Microsoft broke something again, and does not give their customers a good way to fix it.

**Comment by preetriti1** [comment]
@jmancevska @pgferrero - We have identified and fixed this issue, it should be in production by 09/09

**Comment by jmancevska** [comment]
@preetriti1 - We still experience the connection issue today on Sep 10-th.

**Comment by sajid08** [comment]
Still getting this error on Service Bus connection, blob is working fine

**Comment by jmancevska** [comment]
@pgferrero, @simenums, @sajid08,
My team had a working session with the Microsoft technical team, where we shared logs and more details around this error.  The Microsoft team found out what the issue was and provided a workaround for our particular case.
They will be publishing the fix to production hopefully in near future.
I will do another post here, and you will be able to check if your issues got resolved with that fix.
